[bug]: Errror in GUI if try to delete one object

### No existing issues.

- [X] There is no existing issue for my problem.

### Describe the bug

If I want to delete one object an error in gui occured

### To Reproduce

1. Go to objects
2. selected one object
3. try to delete
4. a gui error occurs
This was with the version 6.15.0. With 6.14.1 there was no error

### Expected behavior

No error , if I delete one object

### Screenshots & Logfiles

Error: Minified React error #130; visit https://reactjs.org/docs/error-decoder.html?invariant=130&args[]=undefined&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at Rs (react-dom.production.min.js:303:64)
    at e (react-dom.production.min.js:161:400)
    at wo (react-dom.production.min.js:192:308)
    at Ei (react-dom.production.min.js:294:9)
    at ks (react-dom.production.min.js:280:389)
    at ys (react-dom.production.min.js:280:320)
    at vs (react-dom.production.min.js:280:180)
    at us (react-dom.production.min.js:271:88)
    at ss (react-dom.production.min.js:273:300)
    at Hl (react-dom.production.min.js:127:105)
    at react-dom.production.min.js:267:273
(anonymous) @ console.ts:40
Show 1 more frame
Show less

### Adapter version

6.15.0

### js-controller version

5.0.19

### Node version

18.17.1

### Operating system

Debian Linux

### Additional context

_No response_

Same Probleme


console.ts:40 Error: Minified React error #130; visit https://reactjs.org/docs/error-decoder.html?invariant=130&args[]=undefined&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at Rs (react-dom.production.min.js:303:64)
    at e (react-dom.production.min.js:161:400)
    at wo (react-dom.production.min.js:192:308)
    at Ei (react-dom.production.min.js:294:9)
    at ks (react-dom.production.min.js:280:389)
    at ys (react-dom.production.min.js:280:320)
    at vs (react-dom.production.min.js:280:180)
    at us (react-dom.production.min.js:271:88)
    at ss (react-dom.production.min.js:273:300)
    at Hl (react-dom.production.min.js:127:105)
    at react-dom.production.min.js:267:273

--- a/README.md
+++ b/README.md
@@ -89,6 +89,9 @@ The icons may not be reused in other projects without the proper flaticon licens
 	### **WORK IN PROGRESS**
 -->
 ## Changelog
+### **WORK IN PROGRESS**
+* (foxriver76) fixed issue on deleting admin
+
 ### 6.15.0 (2024-03-01)
 * (foxriver76) harmonized the links to adapter documentation (and fallback to english if no explicit doc for sys language provided)
 * (bluefox) added possibility to delete whole adapter from instance tab if only one instance left

--- a/packages/admin/src/src/components/Adapters/AdapterRow.tsx
+++ b/packages/admin/src/src/components/Adapters/AdapterRow.tsx
@@ -43,7 +43,6 @@ import Link from '@mui/material/Link';
 import MonetizationOn from '@mui/icons-material/MonetizationOn';
 import IsVisible from '../IsVisible';
 import MaterialDynamicIcon from '../../helpers/MaterialDynamicIcon';
-// @ts-expect-error fix later
 import sentryIcon from '../../assets/sentry.svg';
 
 const styles = (theme: Theme) => ({

--- a/packages/admin/src/src/components/Adapters/AdapterTile.tsx
+++ b/packages/admin/src/src/components/Adapters/AdapterTile.tsx
@@ -25,7 +25,6 @@ import { amber } from '@mui/material/colors';
 import { i18n, Utils } from '@iobroker/adapter-react-v5';
 
 import Link from '@mui/material/Link';
-// @ts-expect-error adapt tsconfig
 import sentryIcon from '../../assets/sentry.svg';
 import IsVisible from '../IsVisible';
 

--- a/packages/admin/src/src/dialogs/GitHubInstallDialog.tsx
+++ b/packages/admin/src/src/dialogs/GitHubInstallDialog.tsx
@@ -29,7 +29,6 @@ import {
 
 import { I18n, Icon } from '@iobroker/adapter-react-v5';
 
-// @ts-expect-error adapt tsconfig
 import npmIcon from '../assets/npm.png';
 
 function a11yProps(index: number): {id: string; 'aria-controls': string} {

--- a/packages/admin/src/src/dialogs/WizardDialog.tsx
+++ b/packages/admin/src/src/dialogs/WizardDialog.tsx
@@ -1,6 +1,5 @@
-import { Component } from 'react';
-import { withStyles } from '@mui/styles';
-import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { type Styles, withStyles } from '@mui/styles';
 
 import {
     Button,
@@ -11,7 +10,7 @@ import {
     Toolbar,
     Stepper,
     Step,
-    StepLabel,
+    StepLabel, type Theme,
 } from '@mui/material';
 
 // Icons
@@ -22,7 +21,7 @@ import {
 
 import {
     withWidth, Utils,
-    Router, ToggleThemeMenu,
+    Router, ToggleThemeMenu, I18n, type AdminConnection,
 } from '@iobroker/adapter-react-v5';
 
 import WizardPasswordTab from '../components/Wizard/WizardPasswordTab';
@@ -37,7 +36,7 @@ import LongLogo from '../assets/longLogo.svg';
 
 const TOOLBAR_HEIGHT = 64;
 
-const styles = theme => ({
+const styles = (theme: Theme) => ({
     dialog: {
         height: '100%',
         maxHeight: '100%',
@@ -106,10 +105,27 @@ const styles = theme => ({
     finalLongLogo: {
         width: 500,
     },
-});
+}) satisfies Styles<any, any>;
 
-class WizardDialog extends Component {
-    constructor(props) {
+interface WizardDialogProps {
+    t: typeof I18n.t;
+    socket: AdminConnection;
+    onClose: (redirect?: string) => void;
+    toggleTheme: () => void;
+    themeName: string;
+    classes: Record<string, any>;
+}
+
+interface WizardDialogState {
+    activeStep: number;
+    auth: boolean;
+    secure: boolean;
+}
+
+class WizardDialog extends Component<WizardDialogProps, WizardDialogState> {
+    private adminInstance: null | ioBroker.AdapterObject = null;
+
+    constructor(props: WizardDialogProps) {
         super(props);
 
         this.state = {
@@ -117,12 +133,11 @@ class WizardDialog extends Component {
             auth: false,
             secure: false,
         };
-        this.adminInstance = null;
     }
 
     componentDidMount() {
         this.props.socket.getCurrentInstance()
-            .then(namespace =>
+            .then((namespace: string) =>
                 this.props.socket.getObject(`system.adapter.${namespace}`)
                     .then(obj => {
                         this.adminInstance = obj;
@@ -160,7 +175,7 @@ class WizardDialog extends Component {
             t={this.props.t}
             socket={this.props.socket}
             themeName={this.props.themeName}
-            onDone={settings => {
+            onDone={(settings: any) => {
                 this.props.socket.getSystemConfig(true)
                     .then(obj => {
                         obj.common.licenseConfirmed = true;
@@ -179,7 +194,7 @@ class WizardDialog extends Component {
             t={this.props.t}
             socket={this.props.socket}
             themeName={this.props.themeName}
-            onDone={pass =>
+            onDone={(pass: string) =>
                 this.props.socket.changePassword('admin', pass)
                     .then(() =>
                         this.setState({ activeStep: this.state.activeStep + 1 }))}
@@ -191,7 +206,7 @@ class WizardDialog extends Component {
             t={this.props.t}
             socket={this.props.socket}
             themeName={this.props.themeName}
-            onDone={settings =>
+            onDone={(settings: any) =>
                 this.props.socket.getSystemConfig(true)
                     .then(obj => {
                         Object.assign(obj.common, settings);
@@ -209,7 +224,7 @@ class WizardDialog extends Component {
             secure={this.state.secure}
             socket={this.props.socket}
             themeName={this.props.themeName}
-            onDone={settings =>
+            onDone={(settings: any) =>
                 this.setState(settings, () => this.setState({ activeStep: this.state.activeStep + 1 }))}
         />;
     }
@@ -234,7 +249,7 @@ class WizardDialog extends Component {
             let certPublic;
             let certPrivate;
             if (this.adminInstance.native.secure !== this.state.secure || this.adminInstance.native.auth !== this.state.auth) {
-                if (this.state.secure && (!this.adminInstance.certPublic || !!this.adminInstance.certPrivate)) {
+                if (this.state.secure && (!this.adminInstance.native.certPublic || !!this.adminInstance.native.certPrivate)) {
                     // get certificates
                     try {
                         const certs = await this.props.socket.getCertificates();
@@ -253,6 +268,7 @@ class WizardDialog extends Component {
                         this.adminInstance.native.secure = false;
 
                         await this.props.socket.setObject(this.adminInstance._id, this.adminInstance);
+                        // @ts-expect-error check later
                         setTimeout(() => window.location = `http://${window.location.host}/#tab-adapters${discovery?.val ? '/discovery' : ''}`, 1000);
                         this.props.onClose();
                         return;
@@ -321,7 +337,7 @@ class WizardDialog extends Component {
                 <img src={Logo} className={this.props.classes.logo} alt="logo" />
                 {this.props.t('Initial ioBroker setup')}
                 {' '}
-                <ToggleThemeMenu className={this.props.classes.themeButton} t={this.props.t} toggleTheme={this.props.toggleTheme} themeName={this.props.themeName} size="small" />
+                <ToggleThemeMenu className={this.props.classes.themeButton} t={this.props.t} toggleTheme={this.props.toggleTheme} themeName={this.props.themeName as any} size="small" />
             </DialogTitle>
             <DialogContent className={this.props.classes.content}>
                 <AppBar position="static">
@@ -348,13 +364,5 @@ class WizardDialog extends Component {
         </Dialog>;
     }
 }
-
-WizardDialog.propTypes = {
-    t: PropTypes.func,
-    socket: PropTypes.object,
-    onClose: PropTypes.func,
-    toggleTheme: PropTypes.func,
-    themeName: PropTypes.string,
-};
 
 export default withWidth()(withStyles(styles)(WizardDialog));

--- a/packages/admin/src/src/globals.d.ts
+++ b/packages/admin/src/src/globals.d.ts
@@ -3,8 +3,10 @@ declare global {
         CryptoJS: any;
         _localStorage?: Storage;
     }
-}
 
-declare module '*.svg'
+    declare module '*.svg'
+    declare module '*.png'
+    declare module '*.jpg'
+}
 
 export {};

--- a/packages/admin/src/src/tabs/Objects.jsx
+++ b/packages/admin/src/src/tabs/Objects.jsx
@@ -15,7 +15,7 @@ import {
 
 // icons
 import {
-    Cloas as IconCancel,
+    Cancel as IconCancel,
     Delete as IconDeleteOne,
 } from '@mui/icons-material';
 

